hdimageview/src/main/java/xyz/zpayh/hdimageview/SimpleValueAnimator.java

```diff
@@ -141,7 +141,10 @@ public void run() {
             } else {
 //                HandlerActionQueue handlerActionQueue = new HandlerActionQueue();
```

Comment:
every file has comments which should be removed. Too many unwanted code

Reply:
Yes, Actually this library is not completely implemented still some issues are not resolved.